corehq/apps/users/user_data.py

```diff
@@ -213,4 +258,10 @@ def prime_user_data_caches(users, domain):
                 user_data = UserData({}, user, domain)
             # prime the user.get_user_data cache
             user._user_data_accessors[domain] = user_data
+
+            # prime the user schema data to avoid individual database calls
+            user._schema_fields = schema_fields
```

Comment:
Shouldn't this be
```suggestion
            user_data._schema_fields = schema_fields
```
Would you mind adding a test similar to `test_prime_user_data_caches` to verify that this prevents DB hits?

Reply:
Thanks, addressed in 0c3f622db14a4858871edf2ceae666d6eff61d97